components/tools/OmeroWeb/test/integration/test_api_containers.py

```diff
@@ -178,8 +179,36 @@ def user_screens(self, user1):
         screens.sort(cmp_name_insensitive)
         return screens
 
+    @pytest.mark.parametrize("dtype", ['Plate', 'Image', 'Well',
+                                       'Channel', 'foo'])
+    def test_crud_unsupported(self, user1, dtype):
+        """Test create, update & delete are rejected for unsupported types."""
```

Comment:
I think you should split the test and not have ``post/put/delete`` tests in the same one